### ISSUE TITLE
Consider making the overview a content page

### ISSUE THREAD

**Issue by victorporof** [mozilla/tofino#912]
tofino://overview

Because why not?

**Comment by jsantell** [comment]
Makes sense for consistency -- I don't think there's much actual value in doing so yet unless we're changing it up a bit

**Comment by Mossop** [comment]
Making it a more normal tab could fix #895

**Comment by bgrins** [comment]
Seems like there'd be a fair amount of special casing needed to make it a normal tab - it can't be closed, the back button works differently, you can't navigate / change the URL.  I guess it's like a pinned and readonly tab.

But there's also special casing needed to fix things like #895 as-is, and maybe we'll want pinned tabs eventually anyway.

**Comment by Mossop** [comment]
We're going to have top special case the tab close code regardless (because of #897). My gut says that we'll use less special casing if overview is what the app thinks of as the currently selected tab vs. having no selected tab (or a phantom tab as it is right now)

**Comment by bgrins** [comment]
> we'll use less special casing if overview is what the app thinks of as the currently selected tab vs. having no selected tab

I agree with that, based on the current design.  If we are fairly settled on the UI being a 'tab-like' thing then I'm all for it.

One risk to making it a tab is if the experimentation leads to a design in which is really doesn't look / act like a tab then the implementation might take on extra complexity (i.e. if it turns into a button on the nav bar).
